inc/metabox/extend-metabox/src/settings.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Meta Options build.
+ */
+ import { PluginSidebar, PluginSidebarMoreMenuItem } from '@wordpress/edit-post';
+ const { __ } = wp.i18n;
+ const { Component, Fragment} = wp.element;
+ const { compose } = wp.compose;
+ const { withSelect, withDispatch } = wp.data;
```

Comment:
```suggestion
 const { __ } = wp.i18n;
 const { Component, Fragment} = wp.element;
 const { compose } = wp.compose;
 const { withSelect, withDispatch } = wp.data;
```
Let's use `@wordpress/` instead of `wp.`

Reply:
W need `wp` to access the `data` variable `@wodpress` only user for importing wp related components